sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -453,21 +458,20 @@ def _should_collect_stats(self):
             is_statsbeat_enabled()
             and not get_statsbeat_shutdown()
             and not self._is_stats_exporter()
+            and not self._is_customer_sdkstats_exporter()
             and not self._instrumentation_collection
         )
 
 
     # check to see whether its the case of customer sdkstats collection
     def _should_collect_customer_sdkstats(self):
-        # Import here to avoid circular dependencies
-        from azure.monitor.opentelemetry.exporter.statsbeat._state import get_customer_sdkstats_shutdown
-
         env_value = os.environ.get("APPLICATIONINSIGHTS_SDKSTATS_ENABLED_PREVIEW", "")
         is_customer_sdkstats_enabled = env_value.lower() == "true"
-        # Don't collect customer sdkstats for instrumentation collection or customer sdkstats exporter
+        # Don't collect customer sdkstats for instrumentation collection, sdkstats exporter or customer sdkstats exporter
         return (
             is_customer_sdkstats_enabled
             and not get_customer_sdkstats_shutdown()
+            and not self._is_stats_exporter()
```

Comment:
This is an important change, we do not want to collect cx stats on regular stats exporter.